test/QuadForm/Torsion.jl

```diff
@@ -192,4 +192,56 @@
   gen = genera((0,6), 2^3*3^3*5^2)
   disc = discriminant_group.(gen)
   @test all(T -> length(collect(T)) == order(T), disc)
+
+  # isometry
+
+  L = Zlattice(gram=matrix(ZZ, [[2, -1, 0, 0, 0, 0],[-1, 2, -1, -1, 0, 0],[0, -1, 2, 0, 0, 0],[0, -1, 0, 2, 0, 0],[0, 0, 0, 0, 6, 3],[0, 0, 0, 0, 3, 6]]))
+  T = discriminant_group(L)
+  N, S = normal_form(T)
+  bool, phi = @inferred is_isometric_with_map(T, N)
+  @test bool
+  @test S.map_ab == phi.map_ab
+  _, phi = is_isometric_with_map(T, T)
+  @test phi.map_ab == id_hom(T).map_ab
+  rq, _ = radical_quadratic(T)
+  @test is_isometric_with_map(rq, torsion_quadratic_module(QQ[2;]))[1]
+  Tsub, _ = sub(T, [gens(T)[1]])
+  @test !is_isometric_with_map(T, Tsub)[1]
+  @test_throws ArgumentError is_isometric_with_map(T, rescale(T, 1//2))
+
+  Tsub, _ = sub(T, [2*gens(T)[1], 3*gens(T)[2]])
+  rq, i = radical_quadratic(Tsub)
+  bool, j = @inferred has_complement(i)
+  N = domain(j)
+  M, i1, i2 = orthogonal_sum(cover(rq), cover(N))
+  W = orthogonal_sum(relations(rq), relations(N))[1]
+  T2 = torsion_quadratic_module(M, W, gens = append!(i1.(lift.(gens(rq))), i2.(lift.(gens(N)))), modulus = modulus(rq), modulus_qf = modulus_quadratic_form(rq))
+  @test is_degenerate(T2)
+  bool, phi = @inferred is_isometric_with_map(Tsub, T2)
+  @test bool
+  @test all(a -> phi(i(a)) == T2(i1(lift(a))), gens(rq))
+
+  # anti isometry
+
+  L = orthogonal_sum(root_lattice(:E, 8), root_lattice(:E, 8))[1]
+  agg = automorphism_group_generators(L)
+  for f in agg
+    if isone(f)
+      continue
+    end
+    L1 = invariant_lattice(L, f)
+    L2 = orthogonal_submodule(L, L1)
+    qL1 = discriminant_group(L1)
+    qL2 = discriminant_group(L2)
+    bool, phi = @inferred is_anti_isometric_with_map(qL2, qL1)
+    @test bool
+    @test is_isometric(lll(overlattice(phi)), L)[1]
+  end
+  
+  f = agg[1]
```

Comment:
this relies on a specific ordering of the output of agg ... but that might not stay consistent over time. 
Better hardcode the matrix.